defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-postgresql_support_libpq_version: "9.5.*.pgdg14.04+2"
+postgresql_support_libpq_version: "9.5.*.pgdg*"
```

Comment:
Hm, ok. Why is this a separate install task then?  Seems like this package goes through pretty frequent updates, so if we could bypass having to specify a version for it, that might save us having to update this role each time.